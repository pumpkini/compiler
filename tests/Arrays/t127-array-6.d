
int main() {
    double[] doubles;

    doubles = NewArray(5, double);

    doubles[0] = 6.3;
    doubles[1] = 4.7;
    doubles[2] = 9.5;
    doubles[3] = 12.5;
    doubles[4] = -3.0;

    Print(doubles[4]);
    Print(doubles[3]);
    Print(doubles[2]);
    Print(doubles[0]);
}
