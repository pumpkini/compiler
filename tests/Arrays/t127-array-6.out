-3.00000000
12.50000000
9.50000000
6.30000019
